secant/Util/DatabaseFiles.swift

```diff
@@ -29,27 +33,63 @@ struct DatabaseFiles {
         }
     }
 
+    func cacheDbURL(for network: String) throws -> URL {
```

Comment:
we should avoid using a string when we can actually use a type.

Instead of ` func cacheDbURL(for network: String) throws -> URL` use ` func cacheDbURL(for network: NetworkType) throws -> URL`

Reply:
Actually I used `ZcashNetwork` in the end, that way I can access constants and the appropriate prefix.